### PR TITLE
Enter validate_create() only if self.worker_pods

### PR DESCRIPTION
**Pull request by Anatw**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -2,18 +2,18 @@ default_language_version:
...
 
 repos:
-  - repo: https://github.com/hadialqattan/pycln
-    rev: "v1.1.0"
-    hooks:
-      - id: pycln
+#  - repo: https://github.com/hadialqattan/pycln
+#    rev: "v1.2.5"
+#    hooks:
+#      - id: pycln
```
[the remark this answers is not in the document]
Comment (Anatw) [reply .pre-commit-config.yaml]: ack

**Review by myakove — APPROVED** [review]

**Comment by Anatw** [comment]
\verified

**Review by rnetser — APPROVED** [review]

**Comment by Anatw** [comment]
!verified

**Pull request merged**